composer.json

```diff
@@ -67,5 +52,8 @@
     ],
     "scripts": {
         "post-autoload-dump": "Dflydev\\EmbeddedComposer\\Core\\Script::postAutoloadDump"
+    },
+    "extra": {
+        "nanbando-bundle-class": "Nanbando\\Bundle\\NanbandoBundle"
```

Comment:
would make a grouped key if maybe other things need to be configured and not sure if I would go with array of classes if you  have a multi package e.g.:

```json
"extra": {
        "nanbando": {
                 "bundle-classes": [
                     "Nanbando\\Bundle\\NanbandoBundle"
                 ]
         }
}
```